packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3224,6 +3226,17 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     markNeedsSemanticsUpdate(onlyLocalUpdates: (value != null) == hadValue);
   }
 
+  /// If non-null, sets the [SemanticsNode.isButton] semantic the given value.
```

Comment:
I think this documentation may have a grammar error in it.

Reply:
done